synthetics/browser-test/browser-test-results.rst

```diff
@@ -143,6 +143,7 @@ Using the waterfall chart, you can do the following:
 - Expand the details in a row to show the request and response headers for that resource.
 - Hover over a row of the timeline to view a pop-up message with detailed request timings for that resource.
 - Search resources in a page by keywords in the URL.
+- Follow a direct link to related back-end spans if the same app is :new-page:`instrumented with APM <https://docs.splunk.com/observability/en/synthetics/set-up-synthetics/set-up-synthetics.html#optional-link-synthetic-spans-to-apm-spans>`.
```

Comment:
We might want to use the ref here, not a static link. 

+CC @bhillmer

Reply:
@theletterf I agree. Not sure how to make changes to this branch though.

Reply:
@bhillmer Maintainers (us) are allowed to edit branches in forks as per repo settings:

![Screenshot 2024-05-29 at 18 03 05](https://github.com/splunk/public-o11y-docs/assets/1773616/4cc0ddd1-f05c-4e9d-824d-d932a8816a1d)

Reply:
Oh thank you! I never would have found that!